src/Appium.Net/Appium.Net.csproj

```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net48;net6.0</TargetFrameworks>
-    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```

Comment:
Why was this removed?

Reply:
Removed and added `netstandard2.0`

Reply:
If you mean `licence.txt`, then it was duplicated.